content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -27,10 +27,11 @@ <h3>Objectives</h3>
                 <h3>Kubernetes Deployments</h3>
                 <p>
                 Once you have a running Kubernetes cluster, you can deploy your containerized applications on top of it.
-                To do so, you create a Kubernetes <b>Deployment</b> configuration. The Deployment instructs Kubernetes
-                how to create and update instances of your application. Once you've created a Deployment, the Kubernetes
-                control plane schedules the application instances included in that Deployment to run on individual Nodes in the
-                cluster.
+                To do so, you create a Kubernetes <b>Deployment</b> (a Kubernetes object that declares your intent to
```

Comment:
I am not sure we can term Deployment as an object that declares the user's intent to run application instances. The existing phrasing works fine, IMO, here & I do not see the value add.

Reply:
For any beginner, It was quite confusing so I simplified that.
@sftim agreed on the issue and he did suggested me these changes.
Pls have a look at it.
https://github.com/kubernetes/website/pull/40033

Reply:
@satyampsoni : Please mention that in the description.